integration-tests/common/test_common.go

```diff
@@ -0,0 +1,253 @@
+package common
+
+import (
+	"context"
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	ops "github.com/smartcontractkit/chainlink-starknet/relayer/ops"
+	starknet "github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet"
+	blockchain "github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	ctfClient "github.com/smartcontractkit/chainlink-testing-framework/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"strings"
+	"time"
+)
+
+const (
+	ChainName = "starknet"
+	ChainId   = "devnet"
+	// These are one of the default addresses based on the seed we pass to devnet which is 123
+	defaultWalletAddress = "0x6e3205f9b7c4328f00f718fdecf56ab31acfb3cd6ffeb999dcbac41236ea502"
+	defaultWalletPrivKey = "0xc4da537c1651ddae44867db30d67b366"
+)
+
+var (
+	err error
+)
+
+type Test struct {
+	sc         *StarkNetDevnetClient
+	cc         *ChainlinkClient
+	mockServer *ctfClient.MockserverClient
```

Comment:
What is the mock server used for?

Reply:
i think it's used to mock an EA, but i wasn't sure how it worked